examples/dai/dai.xml

```diff
@@ -46,9 +52,12 @@
                 </ts:origins>
             </ts:attribute-type>
             <ts:transaction>
-                <ts:ethereum function="dai-bridge" contract="dai-bridge">
+                <!-- convert erc20 DAI to native xDAI -->
+                <!-- simply send dai to 0x4aa42145Aa6Ebf72e164C9bBC74fbD3788045016 -->
+                <ts:ethereum function="transfer" contract="dai">
                     <ts:inputs>
-                        <ts:uint256 ref="amount"/>
+                        <ts:address ref="dst"/>
```

Comment:
@James-Sangalli Do you mean for "dst" and "wad" to refer to an attribute like "amount"?

Reply:
Yes, I am borrowing their naming conventions (which are different for some reason)